tests/snapshots/fooStackSnapshot.ts

```diff
@@ -0,0 +1,148 @@
+import { StackScreenProps } from '@react-navigation/stack';
```

Comment:
I know you mentioned in our call that it's confusing to have real examples used in the snapshot test, but personally I find Foo Bar Bazz more confusing for examples. 

Generally when I'm making changes, I like to copy my actual test case. 

Can we keep both tests?

Reply:
Updated this to use more "realistic" names for things.